src/Microsoft.ML.AutoML/AutoMLExperiment/TrialResult.cs

```diff
@@ -16,6 +16,14 @@ public class TrialResult
 
         public ITransformer Model { get; set; }
 
+        /// <summary>
+        /// the loss for current trial, which is smaller the better. This value will be used to fit smart tuners in <see cref="AutoMLExperiment"/>.
+        /// </summary>
+        public double Loss { get; set; }
```

Comment:
>Loss [](http://example.com/codeflow?start=22&length=4)

does the Loss need to have initial value? or it will be always initialized anyway?